openj9.test.load/src/test.load/net/openj9/stf/IdleLoadTest.java

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+* Copyright (c) 2017 IBM Corp.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License 2.0 which accompanies this distribution
+* and is available at http://eclipse.org/legal/epl-2.0 or the Apache License,
+* Version 2.0 which accompanies this distribution and is available at
+* https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the following Secondary
+* Licenses when the conditions for such availability set forth in the
+* Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+* version 2 with the GNU Classpath Exception [1] and GNU General Public License,
+* version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*******************************************************************************/
+
+package net.openj9.stf;
+
+import net.adoptopenjdk.stf.StfException;
+import net.adoptopenjdk.stf.extensions.core.StfCoreExtension;
+import net.adoptopenjdk.stf.extensions.core.StfCoreExtension.Echo;
+import net.adoptopenjdk.stf.plugin.interfaces.StfPluginInterface;
+import net.adoptopenjdk.stf.environment.StfTestArguments;
+import net.adoptopenjdk.stf.processes.ExpectedOutcome;
+import net.adoptopenjdk.stf.processes.definitions.JavaProcessDefinition;
+import net.adoptopenjdk.stf.processes.definitions.LoadTestProcessDefinition;
+import net.adoptopenjdk.stf.processes.definitions.SystemProcessDefinition;
+import net.adoptopenjdk.stf.runner.modes.HelpTextGenerator;
+
+/**
+ * This is a test plugin for Java.util related tests, it runs a workload of 
```

Comment:
This comment doesn't seem right.

Reply:
Changed this to  "This is a test plugin to run the Idle Microbenchmark as a load test which runs  workloads from the test.idle project."